Edition.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HW12
+{
+    public class Edition
+    {
+        public string Name { get; set; }

```

Comment:
Лучше более точное слово для надвания книги
```suggestion
        public string Title { get; set; }

```